Donation-platform/src/main/webapp/trendingScript.js

```diff
@@ -37,6 +39,7 @@ function queryServer() {
         const out = [];
         charities.forEach((charity) => {
           out.push(charity.name);
+          out.push(charity.tags)
```

Comment:
need semicolon

Reply:
oh, didn't catch that because it successfully ran when I tested. will fix, thanks!

Reply:
it ran successfully without a semicolon? :) How is that possible :)